vertical-pod-autoscaler/pkg/recommender/input/history/history_provider.go

```diff
@@ -26,10 +26,22 @@ import (
 	"k8s.io/autoscaler/vertical-pod-autoscaler/pkg/recommender/model"
 )
 
-const (
-	historyLength  = "8d"
-	podLabelPrefix = "pod_label_"
-)
+type PrometheusHistoryProviderConfig struct {
```

Comment:
I have a doubt. Why struct "PrometheusHistoryProviderConfig" and "prometheusHistoryProviderConfig" have almost similar name and similar fields.

Reply:
yes, those are almost the same, and maybe the unexported one could be removed.

Reply:
removed the unexported one